index.js

```diff
@@ -1 +1,117 @@
 // implement your API here
+
+// Import Express and Cors
+const express = require('express');
+const cors = require('cors');
+
+//Import Actions from db file
+const { find, findById, insert, update, remove } = require('./data/db');
+
+// Instantiate Server App and Port
+const app = express();
+const port = 9000;
+
+// Plug extra functionality needed to use req and res body
+app.use(express.json());
+
+// Enable cors so app work for all origins
+app.use(cors());
```

Comment:
I think the cors middleware should precede the expresson.json middleware

Reply:
Noted!